code/sound/audiostr.cpp

```diff
@@ -819,6 +820,11 @@ float AudioStream::Get_Volume()
 	return m_lVolume;
 }
 
+int AudioStream::Get_Duration()
+{
+	return fl2i(1000.0f * m_fileProps.duration);
+}
```

Comment:
Since this is an accessor function, it should return the field directly as a `double`, just like `Get_Volume` returns its own field directly.  Then the conversion to milliseconds should be done in `audiostream_get_duration`.